src/vcf/odb_report.cpp

```diff
@@ -73,6 +73,11 @@ namespace ebi
         if (transaction.has_current()) {
             transaction.commit();
         }
+
+        {
+            odb::core::connection_ptr c{db->connection()};
```

Comment:
on the other hand, the `commit()` and the `execute()` times are 45 / 55...
Well, the execute is making sqlite freeing memory, It may not be immediate.